core/src/main/kotlin/org/eln2/space/Location.kt

```diff
@@ -2,10 +2,28 @@ package org.eln2.space
 
 import kotlin.math.abs
 
+/**
+ * A vector in two-dimensional space.
```

Comment:
This one in particular deals with *Integers*. Please label as such. There will also be a `Vec2d` which does doubles/floats.